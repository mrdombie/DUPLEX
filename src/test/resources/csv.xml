<?xml version="1.0" encoding="UTF-8"?>
<dataset>
	
	<csv_status id="1" status="PENDING"/>
	<csv_status id="2" status="FAILED"/>
	<csv_status id="3" status="HOLDING"/>
	<csv_status id="4" status="PROCESSED"/>

	<csv id="1"/>
	
	<csv_entry entry_id="1" csv_id="1" name="tim" age="21" height="150" status="HOLDING" />				
				
</dataset>