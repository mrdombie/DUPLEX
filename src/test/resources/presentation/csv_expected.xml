<?xml version="1.0" encoding="UTF-8"?>
<dataset>
	
	<!-- 1	Pending
2	Failed
3	Holding
4	Processed -->
	
	<csv_status id="1" status="PENDING"/>
	<csv_status id="2" status="FAILED"/>
	<csv_status id="3" status="HOLDING"/>
	<csv_status id="4" status="PROCESSED"/>
			
	<csv id = "1"/>
		
	<csv_entry entry_id="1" csv_id="1" name="tim" age="22" height="150" status="PROCESSED" />		
	
</dataset>